tensorflow/core/common_runtime/eager/execute.cc

```diff
@@ -1256,7 +1256,7 @@ Status CreateUnshapedOutput(
 #if defined(IS_MOBILE_PLATFORM)
   return errors::Unimplemented(
       "Remote outputs are not available on mobile devices.");
-#else  // !IS_MOBILE_PLATFORM
+#else   // !IS_MOBILE_PLATFORM
```

Comment:
Nit: Please remove this extra space. (The original code has two spaces between `#else` and `//`, which is already correct.) Some for other places.

(Did you use the wrong formatter option for just this file? Other file seems to be fine.)

Reply:
I do not recall this change, maybe an accident. Just change it back